src/components/package/edit-custom/custom-package-edit.js

```diff
@@ -46,12 +45,51 @@ export default class CustomPackageEdit extends Component {
     proxyTypes: PropTypes.object.isRequired
   };
 
+  static getDerivedStateFromProps(nextProps, prevState) {
+    let stateUpdates = {};
+    const { initialValues } = prevState;
+    const {
+      isSelected,
+      name,
+      contentType,
+      customCoverage,
+      proxy,
+      visibilityData,
+      destroy,
+    } = nextProps.model;
+
+    const selectionStatusChanged = isSelected !== initialValues.isSelected;
+
+    if (selectionStatusChanged) {
+      stateUpdates = {
+        initialValues: {
```

Comment:
you can use your  getInitialValuesFromModel() 

Just add the model as an argument